dlp/snippets/deid.py

```diff
@@ -937,6 +937,79 @@ def deidentify_with_replace_infotype(project, item, info_types):
 # [END dlp_deidentify_replace_infotype]
 
 
+# [START dlp_deidentify_simple_word_list]
+def deidentify_with_simple_word_list(
+    project,
+    input_str,
+    info_type,
+    word_list
+):
+    """Uses the Data Loss Prevention API to de-identify sensitive data in a
+      string by matching against custom word list.
+
+    Args:
+        project: The Google Cloud project id to use as a parent resource.
+        input_str: The string to deidentify (will be treated as text).
+        info_type: A string representing info type to look for.
```

Comment:
why is this needed if this is about custom infotypes ?

Reply:
A name to the custom infotype should be provided, If user is providing the "content string", "word_list", the name to the custom infotype should also be provided by user, as the name would be related to the other inputs they provide, is what I think. Yet let me know if this needs a change. Thank you

Reply:
then it's custom_info_type_name ... this comment doesn't make it clear this is a name to be used for the findings produced by the custom infotype ...

Reply:
yeah right, updated! Thank you for pointing that.